www-static/client/world.js

```diff
@@ -139,70 +139,60 @@ CCC.World.receiveMessage = function(e) {
     CCC.World.setConnected(false);
   } else if (mode === 'message') {
     CCC.World.setConnected(true);
-    var dom = CCC.Common.parser.parseFromString(text, 'text/xml');
-    if (dom.getElementsByTagName('parsererror').length) {
-      // Not valid XML, treat as string literal.
-      dom = document.implementation.createDocument(null, 'text');
-      dom.firstChild.appendChild(document.createTextNode(text));
-    }
-    CCC.World.preprocessXml(dom);
-    for (var i = 0, msg; msg = dom.childNodes[i]; i++) {
-      CCC.World.renderMessage(msg);
+    try {
+      var json = JSON.parse(text);
```

Comment:
Same comment about `json` here as in the core code above.  (Except even more so, since `dom` was at least accurate but `json` is incorrect, because the variable contains POJOs not a JSON string.)  I think this function being called `receiveMessage` suggests `msg` might be the best choice of those I suggested earlier.